src/Models/notADatabase.ts

```diff
@@ -0,0 +1,67 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export class OneDumbDatabase {
```

Comment:
Funny, but lets make this a more serious name as eventually this will contain a database ORM